### ISSUE TITLE
Checkbox: aria-checked attribute not set.

### ISSUE THREAD

**Issue by brunano21** [ibm-js/deliteful#605]
The checkbox widget doesn't set the arribute aria-checked when ticking the box.

Tested on its sample page - Buttons.html.

**Comment by wkeese** [comment]
`aria-checked` isn't necessary on `<input type=checked>` nodes.  It's only a "workaround" for when you have `<div>` etc. mimicking a checkbox.   For deliteful, the focus goes to the `<input>` and therefore `aria-checked` should be unnecessary.

**Comment by brunano21** [comment]
According to (this)[https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_checkbox_role], when using the `role=checkbox`, `aria-checked` is needed.

However, doing some tests with JAWS17, removing `role=checkbox` from the `<input>` leaves the widget working, as it should do. It reacts well when checking/unchecking.

So I would suggest to remove `role=checkbox`, since it has an implicit role.
